checker/tests/nulless-conservative-defaults/annotatedfornullness/AnnotatedForNullness.java

```diff
@@ -34,7 +34,7 @@ Object unannotatedFor(Object test) {
 
     @AnnotatedFor("nullness")
     void foo(@Initialized AnnotatedForNullness this) {
-        // Expect an error because conservative defaults are applied to `unannotatedFor` and it
+        // Expect two errors because conservative defaults are applied to `unannotatedFor` and it
```

Comment:
You didn't add a second expected error, so it's unclear why this should now refer to "two errors". Can you clarify?

Reply:
Oh, the two errors are at one line. 
```
/Users/aosenxiong/waterloo/eisop/checker-framework/checker/tests/nulless-conservative-defaults/annotatedfornullness/AnnotatedForNullness.java:40: error: [argument.type.incompatible] incompatible argument for parameter test of AnnotatedForNullness.unannotatedFor.
        unannotatedFor(initializedField);
                       ^
  found   : @UnknownKeyFor Object
  required: @KeyForBottom Object
/Users/aosenxiong/waterloo/eisop/checker-framework/checker/tests/nulless-conservative-defaults/annotatedfornullness/AnnotatedForNullness.java:40: error: [argument.type.incompatible] incompatible argument for parameter test of AnnotatedForNullness.unannotatedFor.
        unannotatedFor(initializedField);
                       ^
  found   : @Initialized Object
  required: @FBCBottom Object

```

Reply:
You need to make that clearer in the description, in particular how this is different from the error in `bar()` below.

Reply:
Okay, I have updated the comment.